packages/compiler-core/__tests__/utils.spec.ts

```diff
@@ -56,14 +56,6 @@ describe('getInnerRange', () => {
     expect(loc2.end.offset).toBe(4)
   })
 
-  test('at end', () => {
-    const loc2 = getInnerRange(loc1, 4)
-    expect(loc2.start.column).toBe(1)
-    expect(loc2.start.line).toBe(2)
-    expect(loc2.start.offset).toBe(4)
-    expect(loc2.end).toEqual(loc1.end)
-  })
-
```

Comment:
Why do you remove this test case?

Reply:
Because all cases using `getInnerRange` function must have a length parameter.

-  packages/compiler-core/src/transforms/transformElement.ts
![impicture_20210929_124102](https://user-images.githubusercontent.com/26522151/135204413-6c610c17-01ae-4484-9f38-167d1011742e.png)
- packages/compiler-core/src/transforms/vFor.ts
![impicture_20210929_124750](https://user-images.githubusercontent.com/26522151/135204989-eee15463-1074-49af-bd2b-4946960d368f.png)

Empty length will make this change more costly, I think it's a little over designed.

Empty length also never been used in two years, see: #65

If necessary, I will revert this test.

Thanks!